discovery-provider/src/api/v1/models/tracks.py

```diff
@@ -90,6 +90,7 @@
         # on `track_full` for more details
         "downloadable": fields.Boolean,
         "play_count": fields.Integer(required=True),
+        "slug": fields.String,
```

Comment:
can we call this field name "permalink" (from the API persp) instead of slug (forrest's ask and to be consistent w/ our friends over at https://developers.soundcloud.com/docs/api/introduction for terminology

Or keep this and add a new "permalink" field that shows up in non-full API endpoints that is composed of `handle/slug` (you could use "extend_track" to accomplish this)

 From the API persp, a common ask has been for someone to super easily go from discovery node API response to canonical audius.co URL

Reply:
Is it consistent with the semantics of a "permalink" though? I guess maybe since the link will always work even if it changes, but it _does_ change

Reply:
I think the slug nomenclature is good, but has a specific meaning that we're capturing here. we want to make API consumption as straight-forward as possible, which definitely warrants the single field `handle/slug` combo.

I'm not necessarily attached to what we call this, let's ask forrest is he has strong opinions about the naming. `link` would be reasonable to me as well. I just think that a lot of folks coming over from the soundcloud API who are expecting us to have similar response might be searching for a field called "permalink"

Reply:
I suppose soundcloud's "permalink" is effectively just as changing as ours, so should be fine.